pkgs/shells/xonsh/default.nix

```diff
@@ -49,7 +65,7 @@ python3Packages.buildPythonApplication rec {
     changelog = "https://github.com/xonsh/xonsh/raw/${version}/CHANGELOG.rst";
     license = licenses.bsd3;
     maintainers = with maintainers; [ spwhitt vrthra ];
-    platforms = platforms.all;
+    platforms = python3Packages.python.meta.platforms;
```

Comment:
platforms can be removed completely since that is already the default
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/mk-python-derivation.nix#L177

Reply:
```suggestion
```